### DISCUSSION TITLE
Only blocked services working, but filtering rules ignored

### DISCUSSION THREAD

**Discussion by Arne1346** [AdguardTeam/AdGuardHome#7088] [Q&A] [answered]
Hi there,

I am running AGH as HomeAssistant (HA) service on my HA server which is set up with Home Assistant Operating System.

HA and AGH are running fine, I cannot see anywhere any error messages. 
My issue is that AGH only blocks DNS requests which relate to blocked services, but does not take the filter rules or custom filter rules into consideration.

Example:
![image](https://github.com/AdguardTeam/AdGuardHome/assets/72160775/5afdecc5-e742-4c03-a2dc-66b25d41409a)

`log.outbrainimg.com` is processed (not blocked) - even if it is on the custom filter list and the checker confirms it to be blocked:
![image](https://github.com/AdguardTeam/AdGuardHome/assets/72160775/f7deb726-0f96-405f-9d72-c9487adebf34)

???
What is wrong?

Thank you,
Arne

**Comment by Lolen10** [comment]
![grafik](https://github.com/AdguardTeam/AdGuardHome/assets/24716055/10fe067d-a7e7-4faa-8627-7783d48be4cc)

Did you enable this option? I had the same issue until I noticed it was disabled.

**Comment by Arne1346** [comment, marked as answer]
Hi,

my issue was coming from HomeAssistant. Here ADH filtering has a seperate entity and it was disabled. Maybe this happened with an update process...

![image](https://github.com/AdguardTeam/AdGuardHome/assets/72160775/2da01b7c-ff14-489c-8078-da9085e4ea30)

After having switched it back on AGH is working as expected again.
